tests/server/test_params.py

```diff
@@ -357,3 +357,29 @@ def test_parse_day_arg(self):
                 self.assertRaises(ValidationFailedException, parse_day_arg, "time")
             with app.test_request_context("/?time=week:20121010"):
                 self.assertRaises(ValidationFailedException, parse_day_arg, "time")
+
+    def test__combine_source_signal_pairs(self):
+        source_signal_pairs = [
+            SourceSignalPair("src1", ["sig1", "sig2"]),
+            SourceSignalPair("src2", "sig1"),
```

Comment:
does the typing https://github.com/cmu-delphi/delphi-epidata/blob/5a2d6b49214d3584eeec7e2e1cbf0ba6b75e0b9a/src/server/_params.py#L82 actually allow a single string?

Reply:
ah, good point. could've sworn it did!